starboard/nplb/posix_compliance/posix_directory_get_next_test.cc

```diff
@@ -188,10 +188,6 @@ TEST(PosixDirectoryGetNextTest, FailureNullEntry) {
   EXPECT_TRUE(closedir(directory) == 0);
 }
 
-TEST(PosixDirectoryGetNextTest, FailureInvalidAndNull) {
```

Comment:
No need to fix, just asking - is the test not valid/useful ? E.g. why do we remove it ?

Reply:
The test there was for if we provide an invalid directory stream to SbDirectoryGetNext, and made more sense when we had defined values for kSbDirectoryInvalid and specific things to test for in each of those cases. This is left behind from the initial writing of Posix tests.
If I try to give readdir_r an invalid directory, for example, a null pointer, the test hangs forever. It's not a useful test IMO because there's nothing we can actually do to modify the behavior. We just need to ensure we are confirming the directory exists before calling readdir_r, which we do in every case anyway.